src/MiniSequenceEndorser.sol

```diff
@@ -535,4 +560,230 @@ contract MiniSequenceEndorser is IEndorser, Owned {
 
   }
 
+  function operationFilter(
+    IEndorser.Operation calldata operation
+  ) external view returns (
+    address _address,
+    bytes32[][] memory topics
+  ) {
+    _address = operation.entrypoint;
+
+    ExecuteCall memory call = _decodeExecuteCall(operation.data);
+
+    if (_address == GUEST_MODULE_V1) {
+      // TODO: There's no possible filter that can detect v1 TxExecuted events.
+    } else if (_address == GUEST_MODULE_V2) {
+      bytes32 digest = keccak256(abi.encode('guest:', call.txs));
```

Comment:
You shouldn't try to detect the guest module transaction, a fancy operator could strip that out. You should always detect the underlying sequence tx.